data/geolocation-cn

```diff
@@ -157,8 +103,8 @@ include:category-game-accelerator-cn
 # IP 定位网站
 include:ipip
 
-# HTTPDNS
-include:category-httpdns
+# category-httpdns-cn is mainly for advertising purpose
+# include:category-httpdns-cn
```

Comment:
Is it a mistake that this line is commented?

Reply:
It is intentionally commented.
Before I made changes to `category-httpdns(-cn)` , the domains listed there all come from github.com/VirgilClyne/GetSomeFries/blob/main/ruleset/HTTPDNS.Block.yaml
(refer to: https://github.com/v2fly/domain-list-community/pull/2001#issue-2061331828 )

Including them into `geolocation-cn` introduced a potential conflict to users' interests, as the policy bypassing domestic domains might be evaluated before rules that block advertising domains.